medallion/common.py

```diff
@@ -141,34 +86,150 @@ def datetime_to_string_stix(dttm):
 
 def datetime_to_float(dttm):
     """Given a datetime instance, return its representation as a float"""
-    # Based on this solution: https://stackoverflow.com/questions/30020988/python3-datetime-timestamp-in-python2
-    if dttm.tzinfo is None:
-        return calendar.timegm(dttm.utctimetuple()) + dttm.microsecond / 1e6
-    else:
-        return (dttm - dt.datetime(1970, 1, 1, tzinfo=pytz.UTC)).total_seconds()
+    return dttm.timestamp()
 
 
 def float_to_datetime(timestamp_float):
     """Given a floating-point number, produce a datetime instance"""
-    return dt.datetime.utcfromtimestamp(timestamp_float)
+    result = dt.datetime.utcfromtimestamp(timestamp_float)
+    result = result.replace(tzinfo=dt.timezone.utc)
+    return result
 
 
 def string_to_datetime(timestamp_string):
```

Comment:
Seems to revert our TZ fix.

Reply:
Yes, but he's reworking the timezone handling in general; maybe we should accept and eventually reintegrate the change?